lib/workers/repository/__fixtures__/master-issue_with_8_PR.txt

```diff
@@ -4,6 +4,7 @@ This issue lists Renovate updates and detected dependencies. Read the [Dependenc
 
 These branches will be created by Renovate only once you click their checkbox below.
 
+ - [ ] <!-- approve-all-pending-prs -->**Approve all pending PRs**
```

Comment:
These buttons are different because they open _all_ PRs in their section. This also means the button should _look_ different, like one of these maybe:

- [ ] 💯 **Approve all pending PRs** 💯 
- [ ] :rotating_light: **Approve all pending PRs** :rotating_light:
- [ ] :warning: **Approve all pending PRs** :warning:

---

Also, can we use emojis on all of our supported platforms?

- GitHub (.com and Enterprise Server)
- GitLab (.com and CE/EE)
- Bitbucket Cloud
- Bitbucket Server
- Azure DevOps
- Gitea

Reply:
@betterPeleg

Reply:
❗

Reply:
let's go with this type of style - 
 ⚠️ **Approve all pending PRs** ⚠️

Reply:
what about **Open all rate-limited PRs** should I add ⚠️ also?

Reply:
yes. same

Reply:
after your suggestion this is the final result:
https://github.com/MaronHatoum/select-all/issues/68
![image](https://user-images.githubusercontent.com/98313426/183376290-133478d0-542a-4730-9b03-a94dc419d1a4.png)

- is this what we expect the dashboard to look like?

Reply:
I think those all boxes should be last instead of first?

@rarkins WDYT?

Reply:
yeah i agree as well. 
we just thought it might be an issue if there are a lot of pending PRs.
but i agree that for the common use its better to appear last

Reply:
this is an example of the boxes appear last, but as @betterPeleg said if we have a lot of pending approval or rate limited the user should scroll down to mark the new boxes.
https://github.com/MaronHatoum/select-all/issues/68

![image](https://user-images.githubusercontent.com/98313426/183391448-b61fdc32-24e0-428e-a3d2-3168fbdda6a3.png)

Reply:
the user should review all listed pr's anyways before checking the all box anyways